assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -22,6 +22,21 @@ angular.module("openshiftConsole")
 
     $scope.projectName = $routeParams.project;
     $scope.sourceURLPattern = SOURCE_URL_PATTERN;
+    $scope.imageName = $routeParams.imageName;
+
+    $scope.breadcrumbs = [
+      {
+        title: $scope.projectName,
```

Comment:
We need to update the title to the project display name under `ProjectsService.get()` in each of these controllers. See

https://github.com/openshift/origin/blob/master/assets/app/scripts/controllers/setLimits.js#L63-L64